client/src/commands/run.ts

```diff
@@ -226,10 +232,10 @@ export async function runTask(
       await session.cancel();
     }
 
-    running = false;
+    setIsExecutingCode(false);
     commands.executeCommand("setContext", "SAS.running", false);
   });
-  session.onLogFn = onLog ?? LogChannelFn;
+  session.onExecutionLogFn = onLog ?? appendExecutionLogFn;
```

Comment:
I guess it's also good to add
```
session.onSessionLogFn = appendSessionLogFn;
```
for task

Reply:
fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/6d6d1e0dd3d0938d427f339603b093b34cf7d2f0